packages/tools/package.json

```diff
@@ -9,9 +9,9 @@
   "license": "MIT",
   "main": "dist/manypkg-tools.cjs.js",
   "dependencies": {
-    "fast-glob": "^3.3.2",
     "jju": "^1.4.0",
-    "js-yaml": "^4.1.0"
+    "js-yaml": "^4.1.0",
+    "tinyglobby": "^0.2.9"
```

Comment:
hm, this change actually touches `@manypkg/tools` and not the CLI itself - that makes me way more worried about this change since `tinyglobby` might slightly differ in some cases from `fast-glob`

we might have to consider releasing v2 of this package some time in the future to support the upcoming Changesets v3 release, so maybe that would be the best time to revisit this? wdyt?

Reply:
Yes, I was gonna raise that this change requires releasing new versions of `@manypkg/tools` and `@manypkg/get-packages`.

This change does lower the dependency count substantially (by the time everything else merges, it would be by about 30%). However I entirely understand that you don't want it rushed, so just let me know how to best support its eventual merge.